src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -306,31 +392,25 @@ def _cancelAcquisition(self, future):
             logging.debug("Acquisition cancelled.")
         return True
 
-    def _generateScanningIndices(self, rep):
+    def _sort_tile_indices_zigzag(self, tile_indices: List[Tuple[int, int]]) -> List[Tuple[int, int]]:
         """
-        Generate the explicit X/Y position of each tile, in the scanning order
-        # Go left/down, with every second line backward:
-        # similar to writing/scanning convention, but move of just one unit
-        # every time.
-        # A-->-->-->--v
-        #             |
-        # v--<--<--<---
-        # |
-        # --->-->-->--Z
-        rep (int, int): X, Y number of tiles
-        return (generator of tuple(int, int)): x/y positions, starting from 0,0
+        Sort the given tile indices in a zigzag scanning order.
```

Comment:
I think it's worth leaving the ascii art, to explain what "zigzag" mean. You can say it's an example for the case of a 5x3 rectangle.